### DISCUSSION TITLE
Next release

### DISCUSSION THREAD

**Discussion by sabirjana** [skfolio/skfolio#76] [General]
Thank you for the excellent library. When should we expect next release and what might be covered in that?
Thanks!

**Comment by HugoDelatte** [comment]
Thank you!
Next release will be this month. 
I will cover enhancements https://github.com/skfolio/skfolio/issues/67, https://github.com/skfolio/skfolio/issues/69, https://github.com/skfolio/skfolio/issues/70, https://github.com/skfolio/skfolio/issues/71, https://github.com/skfolio/skfolio/issues/72
You can follow the advancement in the development branch https://github.com/skfolio/skfolio/tree/development

**Reply by sabirjana** [reply]
Thank you!!
